index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
Actually for games, it is quite important that it is allowed to drift so that it can be used for controls: 

```
A game rotation vector sensor is similar to a rotation vector sensor but not using the geomagnetic field. Therefore the Y axis doesn't point north but instead to some other reference. That reference is allowed to drift by the same order of magnitude as the gyroscope drifts around the Z axis.
```